widgetssdk/src/main/java/com/glia/widgets/call/CallActivity.java

```diff
@@ -221,4 +227,31 @@ private void navigateToSurvey(Survey survey) {
         newIntent.putExtra(GliaWidgets.SURVEY, (Parcelable) survey);
         startActivity(newIntent);
     }
+
+    // TODO create builder class
```

Comment:
Will you process this TODO?

Reply:
WILCO

Reply:
https://github.com/salemove/android-sdk-widgets/pull/318